src/Yasumi/Yasumi.php

```diff
@@ -186,6 +186,44 @@ public static function create($class, $year = null, $locale = self::DEFAULT_LOCA
     }
 
     /**
+     * Create a new holiday provider instance.
+     *
+     * A new holiday provider instance can be created using this function. You can use one of the providers included
```

Comment:
This description needs to be edited since this function is using the 'ISO3166-2' code as the input parameter.